packages/now-routing-utils/src/superstatic.ts

```diff
@@ -49,30 +49,24 @@ export function convertRedirects(
 ): Route[] {
   return redirects.map(r => {
     const { src, segments } = sourceToRegex(r.source);
-    try {
-      const loc = replaceSegments(segments, r.destination, true);
-      const route: Route = {
-        src,
-        headers: { Location: loc },
-        status: r.statusCode || defaultStatus,
-      };
-      return route;
-    } catch (e) {
-      throw new Error('Failed to parse destination: ' + r.destination);
```

Comment:
Do we want to leave this error in case there are other destinations that fail to parse?

Reply:
Good point, I added the try/catch back 👍